kolibri/plugins/learn/assets/src/vue/side-nav/index.vue

```diff
@@ -74,6 +76,28 @@
     font-size: $font-size
     font-weight: 300
     overflow: hidden
+    position: fixed
+    z-index: 2
+    @media screen and (orientation: landscape)
+      height: 100%
+      top: 0
+      left: 0
+      width: $left-margin - $card-gutter * 0.5
+    @media screen and (orientation: portrait)
+      bottom: 0
+      width: 100%
+      height: auto
+      display: table
```

Comment:
Please do. I use table for outer, table-row for middle, table-cell for inner to achieve equally divided arbitrary number of tabs. If there are Jeet way to do this, definitely go for it